Issue #135

pahnin opened the pull request:
Fixed issue 135 and notification on failure of ajax get request in home page

pahnin committed:
Issue 135 homepage refresh
un commented javascript to use partials
commented javascript which refreshes the pages

Also added hint if no builds were added,
no it will show a message to use bin/goldberg to add
projects if no projects are added

pahnin committed:
Issue 135 homepage refresh
un commented javascript to use partials
commented javascript which refreshes the pages

Also added hint if no builds were added,
no it will show a message to use bin/goldberg to add
projects if no projects are added

pahnin committed:
Merge branch 'master' of github.com:pahnin/goldberg
some fixes for my previous push

pahnin committed:
Small error fix, removed the code I added previously,
Issue 135 homepage refresh

pahnin committed:
Fixed the error in appending the  content fetched by ajax
added updating message while doing ajax request

pahnin committed:
.css files were ignored by the .gitignore
so updated css now

pahnin committed:
Fixing errors
revert back .gitignore to hide .css
fixed javascript errors
update failed message on get failure
stop the server after page loads to check this and when server starts it
automatically updates and hides the error

pahnin committed:
deleting css files

srushti commented on app/views/home/_group_projects.html.haml~:
```diff
@@ -0,0 +1,16 @@
```
Please fix your global .gitignore and/or editor. Your editor seems to be generating these '~' files (there's two of them in this pull request). Also, delete these files from this pull request.

srushti commented on public/stylesheets/reset.css:
```diff
@@ -1,156 +0,0 @@
```
This isn't getting generated from anything. We just got this from somewhere else. Our .css files are being generated by us, but not this one. It needs to be restored.

pahnin replied:
I think I removed all the css files in the stylesheets directory! So, whats my best option to restore it?

pahnin replied:
I did the recovery and deleted the ~ files, closing this pull- request and re requesting.
Thanks

pahnin committed:
Recovering reset.css and removing ~ files

I've figured how to recover, with this link
http://stackoverflow.com/questions/953481/restore-a-deleted-file-in-a-git-repo

So, I believe its the same reset.css file, and deleted rest css files in the stylesheet directory.

Also removed ~ files, I'll make sure it'll not be repeated again.
